privacyidea/api/token.py

```diff
@@ -426,9 +426,8 @@ def assign_api():
         err_message = "Token already assigned to another user."
     else:
         err_message = None
-    res = assign_token(serial, user, pin=pin, encrypt_pin=encrypt_pin,
-                       err_message=err_message)
-    g.audit_object.log({"success": True})
+    res = assign_token(serial, user, pin=pin, encrypt_pin=encrypt_pin, err_message=err_message)
+    g.audit_object.log({"success": res})
```

Comment:
{"success": True}

Reply:
Actually, ``assign_token`` either raises an error or returns ``True``, so if this line is executed at all, ``res`` will always be True.

But maybe we should make this more explicit?

Reply:
No problem, did this in dc564c7f3faaaf2ebc6315523cd0cdd481131c74